src/core/symbology-ng/qgsmarkersymbollayerv2.h

```diff
@@ -73,6 +75,9 @@ class CORE_EXPORT QgsSimpleMarkerSymbolLayerV2 : public QgsMarkerSymbolLayerV2
     Qt::PenStyle outlineStyle() const { return mOutlineStyle; }
     void setOutlineStyle( Qt::PenStyle outlineStyle ) { mOutlineStyle = outlineStyle; }
 
+    Qt::PenJoinStyle penJoinStyle() const { return mPenJoinStyle; }
```

Comment:
Don't forget to add the doc strings here  (and setPenJoinStyle ) - that will also trigger a test failure. Make sure you include an `@note added in QGIS 2.16`